nextflow_schema.json

```diff
@@ -539,6 +551,19 @@
                     "default": "bam_stat,inner_distance,infer_experiment,junction_annotation,junction_saturation,read_distribution,read_duplication",
                     "fa_icon": "fas fa-chart-pie",
                     "description": "Specify the RSeQC modules to run."
+                },
+                "kraken_db": {
```

Comment:
I would favor a parameter to explicitly turn this functionality on, rather than running it by default as soon as a reference database is provided. Something along the line of `params.contaminant_screening` that would allow for the values "Kraken2", "Kraken2/Bracken" and "off" or something the like.

Reply:
Working on this now. I'll do `null` for `"off"`

Reply:
I have very little knowledge of Java and Groovy, so it is hard for me to judge potential side effects of using `null`.  I think, in most places of the pipeline, we are using the safe navigation operator ?, so it will probably not throw a `NullPointerException`. 

But I know that there is a lot of gut feeling involved when people write processes and pipelines. Furthermore, Nextflow's error messages are quite enigmatic at times. I think a simple string comparison is more robust (albeit homespun) and easier to validate than allowing for a null value. But I leave that decision to the more experienced developers of this pipeline.

Reply:
It may be a little more robust, but it seems for other optional params (like `pseudoaligner`) `null` is used which was the basis of my thinking

Reply:
Resolved with [102152e](https://github.com/nf-core/rnaseq/pull/1351/commits/102152e5b07b1fa9d7f65c913e388db9bc6bc2c4), used `null`